portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/UpgradeReport.testcase

```diff
@@ -5,28 +5,28 @@ definition {
 	property data.archive.type = "data-archive-portal";
 	property database.types = "mysql";
 	property portal.release = "true";
-	property portal.upstream = "true";
+	property portal.upstream = "lucas";
 	property portal.version = "7.4.13";
-	property testray.main.component.name = "Database Upgrade Framework";
+	property skip.start.app.server = "true";
+	property testcase.url = "http://www.example.com";
+	property testray.main.component.name = "Database Upgrade Tool";
```

Comment:
Shouldn't this be Database Upgrade Tool? Since the Upgrade Report it's only used with the tool, I always interpreted the report as a part of the DB Upgrade Tool